test-debs.sh

```diff
@@ -66,7 +79,9 @@ cat <<-'EOBASH'
 	docker buildx create --name tianon --node tianon --driver docker-container --driver-opt image=tianon/buildkit --bootstrap
 	printf 'FROM hello-world\nRUN ["/hello"]' | docker buildx build --builder tianon --tag hello-build:buildx -
 
-	docker image inspect --format '.' hello-build:classic hello-build:buildkit hello-build:buildx > /dev/null
+	docker buildx build --tag hello-build:buildx-git 'https://github.com/docker-library/hello-world.git#3fb6ebca4163bf5b9cc496ac3e8f11cb1e754aee:amd64/hello-world'
```

Comment:
Sneaking this test in too -- I _thought_ I broke this as well, but in testing it turns out I'd just had a typo and the error message I got was something obscure like "failed to find username" instead of a classic 404 (:upside_down_face:), but it felt worth keeping anyways because it's a good simple smoke test of the Git integration.

Reply:
I guess I should do the same for `docker build` without `buildx`. :thinking: